ts/util/table-builder.ts

```diff
@@ -125,9 +125,10 @@ export class TableBuilder<ParentType, ChildType> {
     this.addPagingControls(topControlsContainer);
     this.container.appendChild(topControlsContainer);
 
+    const tableWrapper = document.createElement("div");
+    tableWrapper.className = "relative";
     const tableContainer = document.createElement("div");
-    tableContainer.className =
-      "mt-4 overflow-x-auto overflow-y-hidden relative";
+    tableContainer.className = "mt-4 overflow-x-auto overflow-x-hidden";
```

Comment:
Accidentally changed `overflow-y-hidden` to `overflow-x-hidden`? Entire page scrolls horizontally to table width now, but the table is also cut off so there's nothing visible in that additional space